### PR TITLE
Corrected docblock on Helper

### PR DESCRIPTION
**Pull request by ekmst**
Hello!

* Type: code quality | documentation
* Link to issue:

**In raising this pull request, I confirm the following:**

- [x] I have read and understood the [Contributing Guidelines](https://github.com/phalcon/cphalcon/blob/master/CONTRIBUTING.md)
- [x] I have checked that another pull request for this purpose does not exist
- [ ] I wrote some tests for this PR
- [ ] I updated the CHANGELOG

Small description of change: Corrected docblock on Helper and tests Helper\Str

Thanks

### CODE REVIEW

File: phalcon/Helper/Arr.zep [hunk 1]
```diff
@@ -202,9 +221,9 @@ class Arr
     /**
      * Sorts a collection of arrays or objects by key
      *
-     * @param array $collection
-     * @param [type] $attr
-     * @param [type] $order
+     * @param array  $collection
+     * @param mixe   $attr
```
Comment (niden) [inline phalcon/Helper/Arr.zep]: mixed :)

**Review by niden — CHANGES_REQUESTED** [review]

**Comment by niden** [comment]
One minor change. Thank you @ekmst you rock!

File: phalcon/Helper/Arr.zep (hunk 1 above)
Comment (ekmst) [reply phalcon/Helper/Arr.zep]: fixed :)

**Review by niden — APPROVED** [review]

**Comment by niden** [comment]
Thank you @ekmst

**Pull request merged**